src/test/java/com/mael/ttt/GameTest.java

```diff
@@ -57,7 +62,7 @@ public void runEndsTheGameIfFull() {
     public void checkForFullPrintsBoardAndMsgs() {
         spy.setInputs("1", "1", "2", "3", "7", "4", "6", "5", "9", "8", "n");
         game.run();
-        assertThat(spy.printedMessage(), containsString("X O X \nX X O \nO X O \n" + UserInterface.ISFULL + UserInterface.GAMEOVER));
+        assertThat(spy.printedMessage(), containsString(playerMark + " " + opponentMark + " " + playerMark + " \n" + playerMark + " " + playerMark + " " + opponentMark + " \n" + opponentMark + " " + playerMark + " " + opponentMark + " \n" + UserInterface.ISFULL + UserInterface.GAMEOVER));
```

Comment:
I liked the simple string you had before :crying_cat_face:  :smile: Perhaps you could extract this into a method to make it look simpler?